sympy/printing/tests/test_lambdarepr.py

```diff
@@ -145,5 +147,36 @@ def test_piecewise():
         "(True) else None)))))))))))"
 
 
+def test_sum():
+    # In each case, test eval() the lambdarepr() to make sure that
+    # it evaluates to the same results as the symbolic expression
+
+    s = Sum(x ** i, (i, a, b))
+
+    l = lambdarepr(s)
+    assert l == "(builtins.sum(x**i for i in range(a, b+1)))"
+
+    assert (lambdify((x, a, b), s)(2, 3, 8) ==
+            s.subs([(x, 2), (a, 3), (b, 8)]).doit())
+
+    s = Sum(i * x, (i, a, b))
+
+    l = lambdarepr(s)
+    assert l == "(builtins.sum(i*x for i in range(a, b+1)))"
+
+    assert (lambdify((x, a, b), s)(2, 3, 8) ==
+            s.subs([(x, 2), (a, 3), (b, 8)]).doit())
+
+
+def test_multiple_sums():
+    s = Sum(i * x + j, (i, a, b), (j, c, d))
+
+    l = lambdarepr(s)
+    assert l == "(builtins.sum((builtins.sum(i*x + j for i in range(a, b+1))) " \
```

Comment:
`sum(i*x + j for i in range(a, b + 1) for j in range(c, d + 1))` should be the same but more efficient.